### PR TITLE
[WIP] DO NOT MERGE, trying something

### PR DESCRIPTION
**Pull request by aliok**

### CODE REVIEW

File: openshift/e2e-common.sh [hunk 1]
```diff
@@ -127,7 +125,8 @@ function run_e2e_tests(){
 
   go_test_e2e -timeout=90m -parallel=12 ./test/e2e \
     "$run_command" \
-    $common_opts --dockerrepo "quay.io/openshift-knative" || failed=$?
+    $common_opts --dockerrepo "quay.io/openshift-knative" --tag "v0.17" || failed=$?
```
Comment (matzew) [inline openshift/e2e-common.sh:128]: +1

**Comment by aliok** [comment]
/retest

Previous stuff that failed doesn't fail anymore.

`TestBrokerRedelivery` fails, which might be flake

**Comment by aliok** [comment]
/close

Closing in favor of https://github.com/openshift/knative-eventing-contrib/pull/582

**Pull request closed without merging**